src/components/v5/shared/MembersAvatars/MembersAvatars.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Id } from '@colony/colony-js';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import {
+  calculateLastSliceIndex,
+  calculateRemainingItems,
+} from '~utils/avatars';
+import { notNull } from '~utils/arrays';
+import UserAvatar from '~v5/shared/UserAvatar';
+import { SpinnerLoader } from '~shared/Preloaders';
+import { MembersAvatarsProps } from './types';
+import { Watcher } from '~types';
+
+const displayName =
+  'v5.common.ActionSidebar.partials.motions.Motion.steps.OutcomeStep.partials.MembersAvatars';
+
+function MembersAvatars<TValue extends Watcher>({
```

Comment:
This component is not reusable. As a prop it should have a list of items and not watchers. Please remove all of the redundant logic.